docs/intro/ff-ui/dashboard.md

```diff
@@ -0,0 +1,109 @@
+---
+slug: /flutterflow-ui/dashboard
+title: Dashboard
+description: Explore the dashboard in FlutterFlow, a centralized location for managing projects and account.
+tags: [FlutterFlow, UI, Design]
+sidebar_position: 0
+keywords: [Dashboard, FlutterFlow, UI, Design]
+---
+# Dashboard
+The first thing you see when you log in to FlutterFlow is the *Dashboard*. It provides a centralized location for managing projects, such as creating new ones, searching, deleting, and duplicating project. In addition, the dashboard allows you to set the desired theme, whether dark or light, for easy viewing.
+
+The dashboard enables easy access to organizational resources, enabling team members to collaborate and work together seamlessly. Furthermore, the dashboard is integrated with a marketplace, allowing users to browse and download widgets, templates, and plugins.
+
+You can also get links to various resources to help you build apps using FlutterFlow. Your account information and plan details are also accessible from this page.
+
+![dashboard](imgs/dashboard.avif)
+
+## Projects
+
+This section displays the projects you have created in FlutterFlow. You can easily navigate to any of them from here. Use the overflow menu to rename, duplicate, delete, leave the project, add tags, and open the project in a new browser tab.
+
+:::info
+When you duplicate a project containing the Firebase setup, you must delete the config files in your duplicated project and complete a new [**Firebase setup**](../../ff-integrations/firebase/connect-to-firebase-setup.md) for your project.
+:::
+
+## Dark/Light mode switch
```

Comment:
All of these points should be in single bullet points instead. There is too much of wasted space. And more scrolling is required to compare text and image.

Reply:
Agree! done.